File: app/src/main/java/com/rdev/tryp/model/ride_responce/RideRequest.java
package com.rdev.tryp.model.ride_responce;

import com.google.gson.annotations.SerializedName;

public class RideRequest {

    @SerializedName("is_asap")
    private int isAsap;

    @SerializedName("pickup_lng")
    private String pickupLng;

    @SerializedName("destination_lng")
    private String destinationLng;

    @SerializedName("driver_id")
    private String driverId;

    @SerializedName("destination_address")
    private String destinationAddress;

    @SerializedName("user_id")
    private String userId;

    @SerializedName("destination_lat")
    private String destinationLat;

    @SerializedName("pickup_address")
    private String pickupAddress;

    @SerializedName("pickup_time")
    private String pickupTime;

    @SerializedName("ip_address")
    private String ipAddress;

    @SerializedName("pickup_lat")
    private String pickupLat;

    @SerializedName("ride_request_id")
    private String requestId;

    public void setIsAsap(int isAsap) {
        this.isAsap = isAsap;
    }

    public int getIsAsap() {
        return isAsap;
    }

    public void setPickupLng(String pickupLng) {
        this.pickupLng = pickupLng;
    }

    public String getPickupLng() {
        return pickupLng;
    }

    public void setDestinationLng(String destinationLng) {
        this.destinationLng = destinationLng;
    }

    public String getDestinationLng() {
        return destinationLng;
    }

    public void setDriverId(String driverId) {
        this.driverId = driverId;
    }

    public String getDriverId() {
        return driverId;
    }

    public void setDestinationAddress(String destinationAddress) {
        this.destinationAddress = destinationAddress;
    }

    public String getDestinationAddress() {
        return destinationAddress;
    }

    public void setUserId(String userId) {
        this.userId = userId;
    }

    public String getUserId() {
        return userId;
    }

    public void setDestinationLat(String destinationLat) {
        this.destinationLat = destinationLat;
    }

    public String getDestinationLat() {
        return destinationLat;
    }

    public void setPickupAddress(String pickupAddress) {
        this.pickupAddress = pickupAddress;
    }

    public String getPickupAddress() {
        return pickupAddress;
    }

    public void setPickupTime(String pickupTime) {
        this.pickupTime = pickupTime;
    }

    public String getPickupTime() {
        return pickupTime;
    }

    public void setIpAddress(String ipAddress) {
        this.ipAddress = ipAddress;
    }

    public String getIpAddress() {
        return ipAddress;
    }

    public void setPickupLat(String pickupLat) {
        this.pickupLat = pickupLat;
    }

    public String getPickupLat() {
        return pickupLat;
    }

    public void setRequestId(String requestId) {
        this.requestId = requestId;
    }

    public String getRequestId() {
        return requestId;
    }

    @Override
    public String toString() {
        return
                "RideRequest{" +
                        "is_asap = '" + isAsap + '\'' +
                        ",pickup_lng = '" + pickupLng + '\'' +
                        ",destination_lng = '" + destinationLng + '\'' +
                        ",driver_id = '" + driverId + '\'' +
                        ",destination_address = '" + destinationAddress + '\'' +
                        ",user_id = '" + userId + '\'' +
                        ",destination_lat = '" + destinationLat + '\'' +
                        ",pickup_address = '" + pickupAddress + '\'' +
                        ",pickup_time = '" + pickupTime + '\'' +
                        ",ip_address = '" + ipAddress + '\'' +
                        ",pickup_lat = '" + pickupLat + '\'' +
                        ",request_id = '" + requestId + '\'' +
                        "}";
    }
}

File: app/src/main/java/com/rdev/tryp/model/ride_responce/RideRequest.kt
package com.rdev.tryp.model.ride_responce

import com.google.gson.annotations.SerializedName


class RideRequest {

    @SerializedName("is_asap")
    var isAsap: Int = 0

    @SerializedName("pickup_lng")
    var pickupLng: String? = null

    @SerializedName("destination_lng")
    var destinationLng: String? = null

    @SerializedName("driver_id")
    var driverId: String? = null

    @SerializedName("destination_address")
    var destinationAddress: String? = null

    @SerializedName("user_id")
    var userId: String? = null

    @SerializedName("destination_lat")
    var destinationLat: String? = null

    @SerializedName("pickup_address")
    var pickupAddress: String? = null

    @SerializedName("pickup_time")
    var pickupTime: String? = null

    @SerializedName("ip_address")
    var ipAddress: String? = null

    @SerializedName("pickup_lat")
    var pickupLat: String? = null

    @SerializedName("ride_request_id")
    var requestId: String? = null

    override fun toString(): String {
        return "RideRequest{" +
                "is_asap = '" + isAsap + '\''.toString() +
                ",pickup_lng = '" + pickupLng + '\''.toString() +
                ",destination_lng = '" + destinationLng + '\''.toString() +
                ",driver_id = '" + driverId + '\''.toString() +
                ",destination_address = '" + destinationAddress + '\''.toString() +
                ",user_id = '" + userId + '\''.toString() +
                ",destination_lat = '" + destinationLat + '\''.toString() +
                ",pickup_address = '" + pickupAddress + '\''.toString() +
                ",pickup_time = '" + pickupTime + '\''.toString() +
                ",ip_address = '" + ipAddress + '\''.toString() +
                ",pickup_lat = '" + pickupLat + '\''.toString() +
                ",request_id = '" + requestId + '\''.toString() +
                "}"
    }

}